octobot_tentacles_manager/models/tentacle_package.py

```diff
@@ -43,6 +45,17 @@ def add_artifact(self, new_artifact: artifact.Artifact) -> None:
             self.name = f"{self.artifacts[0].origin_package}{TentaclePackage.ARTIFACT_VERSION_SEPARATOR}" \
                         f"{self.version}_{self.ARTIFACT_SUFFIX}"
 
+    def parse_path_in_name_if_any(self) -> None:
+        """
+        Parse path in package name if any
+        :return: None
+        """
+        if self.name \
+                and len(os.path.split(self.name)) > 1 \
+                and os.path.split(self.name)[0]:
+            self.output_path = os.path.split(self.name)[0]
+            self.name = os.path.split(self.name)[1]
```

Comment:
Should we save `os.path.split(self.name)` outputs to avoid re-computing it each time ?

Reply:
Yes 👍